bin/jobs_functions.js

```diff
@@ -76,8 +77,8 @@ module.exports = {
                 printModuleHelp();
                 break;
             default:
-                util.error("Unknown operatation");
-                break;
+                util.error("Unknown operation");
```

Comment:
This is a recurring pattern, maybe a function "exitWithStatusAndErrorMessage" would save a few lines in the code.